docs/src/_guide/installation-guide.md

```diff
@@ -0,0 +1,237 @@
+
+```bash
+meltano --version
+```
+
+### Next Steps
+
+Now that you've installed Meltano and its requirements, you can continue setting up your Meltano project by following the [Getting Started guide](/getting-started#create-your-meltano-project).
+
+## Docker
+
+[Docker](https://www.docker.com/) is an alternative installation option to [using a virtual environment to run Meltano](#virtual-environment). To use these instructions you will need to [install Docker](https://docs.docker.com/install/) onto your computer and have it running when you execute the commands below.
+
+### Using Pre-built Docker Images
+
+We maintain the [`meltano/meltano`](https://hub.docker.com/r/meltano/meltano) Docker image on Docker Hub, which comes with Python and Meltano pre-installed.
+
+To get the latest version of Meltano, pull the `latest` tag. Images for specific versions of Meltano are tagged `v<X.Y.Z>`, e.g. `v1.55.0`.
+
+By default, these images come with a version of Python chosen for a balance of stability and compatibility, currently Python 3.9.
+If you'd like to use a different version of Python instead, add a `-python<X.Y>` suffix to the image tag, e.g. `latest-python3.7` and `v1.54.0-python3.7`.
+
+```bash
+# download or update to the latest version
+docker pull meltano/meltano
+
+# Or choose a specific version of Meltano and/or Python:
+# docker pull meltano/meltano:v1.55.0
+# docker pull meltano/meltano:latest-python3.7
+# docker pull meltano/meltano:v1.55.0-python3.8
+
+# check the currently installed version
+docker run meltano/meltano --version
+```
+
+### Initialize Your Project
+
+Once you have Docker installed, running, and have pulled the pre-built image you can use Meltano just as you would in our [Getting Started Guide](/getting-started). However, the command line syntax is slightly different. For example, let's create a new Meltano project:
+
+```bash
+cd /your/projects/directory
+
+docker run -v "$(pwd)":/projects \
+             -w /projects \
+             meltano/meltano init yourprojectname
+```
+
+Then you can `cd` into your new project:
+
+```bash
+cd yourprojectname
+```
+
+We can then start the Meltano UI. Since `ui` is the default command, we can omit it.
+
+```bash
+docker run -v "$(pwd)":/project \
+             -w /project \
+             -p 5000:5000 \
+             meltano/meltano
+```
+
+You can now visit [http://localhost:5000](http://localhost:5000) to access the Meltano UI.
+
+Now that you're successfully running Meltano, you can continue setting up your Meltano project by following the [Getting Started guide](/getting-started).
+
+Note that wherever you are asked to run the `meltano` command, you will want to run it through `docker run` as in the snippet above.
+
+
+# Troubleshooting Installation
+
+<div class="notification is-info">
+  <p>Are you having installation or deployment problems? We are here to help you. Check out <a href="/">Getting Help</a> on the different ways to get in touch with us.</p>
```

Comment:
Why does the "Getting Help" link send users to the root of the doc site? Can we use some tag to have the page scroll down to the "Get Help" section at the bottom of the page at the root of the doc site?

Reply:
@WillDaSilva I agree with almost all of it, I'll take a look at it. 

I do still like to have the installation separated into two parts though, 

1. basic installation (used in the Tutorial)
2. advanced installation 

The idea is that the intended reader are two different people.
1. is for the first time meltano user, and I want him to get started as fast as possible, so he shouldn't even try to install pipx if he hasn't, or mount the docker volume.

Once someone has gone through the tutorial, he is ready for (2). But in that case, I don't want to bother him with the short version. 

Maybe we just need to distinguish the two better? By name & title? Do you think that would work for you?